misc/python/materialize/mzcompose/services.py

```diff
@@ -970,3 +975,38 @@ def __init__(
                 "volumes": [f"{config.name}:/root/.config/mz/profiles.toml"],
             },
         )
+
+
+class Prometheus(Service):
+    def __init__(self, name: str = "prometheus") -> None:
+        super().__init__(
+            name=name,
+            config={
+                "image": "prom/prometheus:v2.41.0",
+                "ports": ["9090"],
+                "volumes": [
+                    str(ROOT / "misc" / "mzcompose" / "prometheus" / "prometheus.yml")
```

Comment:
Does this work? In other places we've had to use `relpath` with `loader.composition_path`: https://github.com/philip-stoev/materialize/blob/23a69843e22ae174ebe124a35667a6295aab215e/misc/python/materialize/mzcompose/services.py#L482-L485